dotnet/samples/GettingStarted/Agents/Agent_Step03b_UsingFunctionTools_FromOpenAPI/Program.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+// This sample demonstrates how to use a ChatClientAgent with function tools provided via an OpenAPI spec.
+// It uses functionality from Semantic Kernel to parse the OpenAPI spec and create function tools to use with the Agent Framework Agent.
+
+using Azure.AI.OpenAI;
+using Azure.Identity;
+using Microsoft.Agents.AI;
+using Microsoft.Extensions.AI;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Plugins.OpenApi;
+using OpenAI;
+
+var endpoint = Environment.GetEnvironmentVariable("AZURE_OPENAI_ENDPOINT") ?? throw new InvalidOperationException("AZURE_OPENAI_ENDPOINT is not set.");
+var deploymentName = Environment.GetEnvironmentVariable("AZURE_OPENAI_DEPLOYMENT_NAME") ?? "gpt-4o-mini";
+
+// Load the OpenAPI Spec from a file.
+string fileName = "OpenAPISpec.json";
+using FileStream schemaStream = File.OpenRead(fileName);
+var plugin = await OpenApiKernelPluginFactory.CreateFromOpenApiAsync("github", schemaStream);
```

Comment:
There's a overload that accepts filePath.

```suggestion
var plugin = await OpenApiKernelPluginFactory.CreateFromOpenApiAsync("github", fileName);
```

Reply:
Thanks, updated.